tests/test_scheduler_additional_coverage.py

```
@@ -18,6 +19,28 @@
+    asyncio.set_event_loop(loop)
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(autouse=True)
+async def cleanup_scheduler():
+    """Ensure scheduler is stopped after each test."""
+    yield
+    try:
+        await stop_background_updates()
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)